### PR TITLE
K8SPSMDB-1209 adding a new parameter for arm64 support

### PR DESCRIPTION
**Pull request by ptankov**
Test runs:
https://cloud.cd.percona.com/view/PSMDB/job/psmdb-operator-gke-latest/625/
https://cloud.cd.percona.com/view/PSMDB/job/psmdb-operator-gke-version/803/

### CODE REVIEW

File: cloud/jenkins/psmdb_operator_gke_latest-arm64.groovy [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+    }
+
+    if ("$PLATFORM_VER" == "latest") {
+        USED_PLATFORM_VER = sh(script: "gcloud container get-server-config --region=$region --flatten=channels --filter='channels.channel=RAPID' --format='value(channels.defaultVersion)' | cut -d- -f1", , returnStdout: true).trim()
+    } else {
+        USED_PLATFORM_VER="$PLATFORM_VER"
+    }
+    echo "USED_PLATFORM_VER=$USED_PLATFORM_VER"
+
+    if ("$ARCH" == "amd64") {
```
[the remark this answers is not in the document]
Comment (ptankov) [reply cloud/jenkins/psmdb_operator_gke_latest-arm64.groovy:99]: This file `psmdb_operator_gke_latest-arm64.groovy` is temporary. I will merge it into `psmdb_operator_gke_latest.groovy` eventually. I just didn't want to interrupt the running of the e2e tests for the release, while I was developing it.

[the remark this answers is not in the document]
Comment (ptankov) [reply cloud/jenkins/psmdb_operator_gke_latest-arm64.groovy:99]: Actually, now that we're done running the tests, I can merge it already. Hold on...

**Review by egegunes — APPROVED** [review]

File: cloud/jenkins/psmdb_operator_gke_version.groovy [hunk 2]
```diff
@@ -96,16 +96,24 @@ EOF
     }
     echo "USED_PLATFORM_VER=$USED_PLATFORM_VER"
 
+    if ("$ARCH" == "amd64") {
```
Comment (eleo007) [inline cloud/jenkins/psmdb_operator_gke_version.groovy:99]: Can we make it with backward compatibility? If arch is arm64, else use amd one?

File: cloud/jenkins/psmdb_operator_gke_latest.groovy [hunk 3]
```diff
@@ -96,16 +96,24 @@ EOF
     }
     echo "USED_PLATFORM_VER=$USED_PLATFORM_VER"
 
+    if ("$ARCH" == "amd64") {
```
Comment (eleo007) [inline cloud/jenkins/psmdb_operator_gke_latest.groovy:99]: Can we make it with backward compatibility? If arch is arm64 - use t2a-standard-4, else use amd one?

**Review by eleo007 — APPROVED** [review]

**Pull request merged**